packages/ui/src/components/EventsCard/EventsCard.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import { Box, BoxProps } from '../Box';
+import { Button } from '../Button';
+import { Link } from '../Link';
+import { Typography } from '../Typography';
+import { BUTTON_TEXT, MAIN_TITLE } from './EventsCard.mocks';
+
+export type Event = {
+  id: number;
+  title: string;
+  date: Date | { from: Date; to: Date };
+  description: string;
+  url: string;
+};
+
+export type EventCardProps = { events: Event[] } & BoxProps;
+
+const addZeroToDateValue = (dateValue: number): string => {
+  if (dateValue === undefined) {
```

Comment:
❓ If you made `dateValue` a required parameter why do you need to check it for `undefined`?

Reply:
I think, even is this value is required we can't be sure that this value will not to be undefined.  Date.getDate() can return undefined so this condition is additional protection, but with your suggestion I used padStart so this function is no need.